test/tools/auth0/handlers/actions.tests.js

```diff
@@ -127,7 +127,8 @@ describe('#actions handler', () => {
             expect(params.id).to.equal(actionId);
             return Promise.resolve({ ...action, id: actionId });
           },
-          create: (data) => {
+          create: function(data) {
+            (() => expect(this).to.not.be.undefined)();
```

Comment:
Excuse the IIFE, it was the only way to quell the mass amount of `Expected an assignment or function call and instead saw an expression` linting errors.